src/components/nav/nav.js

```diff
@@ -2,10 +2,12 @@ import './nav.css'
 import {Link} from 'react-router-dom'
 import { useCart } from '../../filter-context/cart-context'
 import { useAuth } from '../../filter-context/auth-context'
+import { useWishlist } from '../../filter-context/wishlist-context'
```

Comment:
same here also can make single index file

Reply:
ok got it